dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -43,11 +45,11 @@ RUN sed -i '/^#.* en_US.* /s/^#//' /etc/locale.gen
 RUN locale-gen
 
 # Install up-to-date version of pip
-RUN curl https://bootstrap.pypa.io/get-pip.py | python3.6
+RUN pip3 install --no-cache-dir setuptools  wheel
```

Comment:
Please install pip from the internet.

Reply:
I feel it is better we stick to the packages provided by the distribution.